Extended Update INSTALL_32.md

ergon-gayko-philipp opened the pull request:
We have realised that an adjustment is still required for the installation of node.js

ergon-gayko-philipp committed:
Extended Update INSTALL_32.md

We have realised that an adjustment is still required for the installation of node.js

ergon-gayko-philipp committed:
Update INSTALL_32.md

Also install specific python3

ergon-gayko-philipp committed:
Update INSTALL_32.md

Also install pip

big-r81 commented:
Hey,

this is a very old branch, which you want to update. Is this itentional?

ergon-gayko-philipp committed:
Update INSTALL_32.md

Added cygwin installation command

pgayko commented:
Hi,
We are trying to make a build to install CouchDB on Win 7, 32-bit. Is there a newer branch we can use to do this?

ergon-gayko-philipp committed:
Update INSTALL_32.md

Add stepp install Windows SDK 8.1

ergon-gayko-philipp committed:
Update INSTALL_32.md

install apt-cyg instead of cyg-get since this cannot be found in choco.extension 1.4

ergon-gayko-philipp committed:
Update INSTALL_32.md

Hint to alternative program path

ergon-gayko-philipp committed:
Update INSTALL_32.md

checkout the right couch-db branch

ergon-gayko-philipp committed:
Update INSTALL_32.md

aria2c needs additional arguments for proper downloads

ergon-gayko-philipp committed:
Update INSTALL_32.md

Add additional paths in VS

ergon-gayko-philipp committed:
Update INSTALL_32.md

Build with Visual Studio instead

ergon-gayko-philipp committed:
Update INSTALL_32.md

Belongs to last commit

ergon-gayko-philipp committed:
Update INSTALL_32.md

Add a description to sym link the VC tools for Erlang build required

ergon-gayko-philipp committed:
Update INSTALL_32.md

belongs to last commit

ergon-gayko-philipp committed:
Update INSTALL_32.md

Add hint to install more compilers in cygwin. Removed custom linking cl nmake and link

ergon-gayko-philipp committed:
Update INSTALL_32.md

Add Windows 8.1 SDK installation step

ergon-gayko-philipp committed:
Update INSTALL_32.md

Hint modify shell_32.sh with correct path

ergon-gayko-philipp committed:
Update INSTALL_32.md

ergon-gayko-philipp committed:
Update INSTALL_32.md

ergon-gayko-philipp committed:
Update INSTALL_32.md

ergon-gayko-philipp committed:
Update INSTALL_32.md

Instructions adapted so that a Win32 CouchDB 2.0.x installer can be built in a Windows 10 Virtualbox VM (as of 2024/10).
